### ISSUE TITLE
[UI Improvement] Tab coloring is not correct and visually blur together

### ISSUE THREAD

**Issue by j1xwnbsr** [ForkIssues/TrackerWin#1804]
There is no visual distinction between separate non-active tabs; there needs to hairlines or visual spacing to make it clear there are other tabs.

The tab coloring does not work as expected; a dot means nothing esp. when there is no visual distinction. Selecting a color should change the color of the tab itself, not add a dot.

Support for multiple rows of tabs would be nice, too, for those of us with a dozen or more projects in play at once.

**Comment by pixiekat** [comment]
> Support for multiple rows of tabs

Yes, yes, a thousand times yes.

**Comment by DanPristupov** [comment]
> > Support for multiple rows of tabs would be nice, too, for those of us with a dozen or more projects in play at once.
> Yes, yes, a thousand times yes.

Try to use workspaces (the last button on the toolbar).

**Comment by j1xwnbsr** [comment]
> Try to use workspaces (the last button on the toolbar).

Yes, but switching contexts like that is a roadblock, so it works best as 'long term' grouping, not as a daily workflow.

**Comment by bergamin** [comment]
> Try to use workspaces (the last button on the toolbar).

Oh I do... I have a workspace for each team I'm the reviewer and one of them have 27 repositories :)

![image](https://user-images.githubusercontent.com/14339291/219386807-f2ee6a3f-16ca-4489-b431-b4c1c45af216.png)

Just hidden the ones you could read a word on them, but left some in so you can see what the issue is

For the record, counting workspaces that aren't directly attached to a team, I have (coincidentally) 27 workspaces set up. 3 of them are in this situation

**Comment by DanPristupov** [comment]
@bergamin may be we can try to make the tab control scrollable

**Comment by bergamin** [comment]
@DanPristupov, both options would help. But if we are going to add scrollable tabbing, I think it would still be useful to have 2 lines of tabs in the same way IntelliJ does it. What they have is the ability to pin tabs. These pinned tabs would create a new line of tabs on top of the usual tabbing

![image](https://user-images.githubusercontent.com/14339291/219392425-368b0516-7521-475a-9ac9-7c00d0a0c819.png)

Actually, this would be very useful because some of these repos receive much more code than others and I could then pin the main ones so they are always visible even when scrolling the other ones

Also an extra could be a way to switch between repos inside that workspace by searching for them on a keyboard shortcut search popup (similar to Ctrl+Shift+N in IntelliJ or Ctrl+P in VSCode or Ctrl+Shift+R in Eclipse)
